prac_05/colour_names.py

```diff
@@ -0,0 +1,14 @@
+COLOUR = {"AliceBlue": "#f0f8ff", "AntiqueWhite": "#faebd7", "AntiqueWhite1": "#ffefdb", "AntiqueWhite2": "#eedfcc",
```

Comment:
should be a plural (COLOURS) as lists tuples and dictionaries hold multiple values